javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javaparsermodel/contexts/TryWithResourceContext.java

```diff
@@ -56,7 +56,7 @@ public TryWithResourceContext(TryStmt wrappedNode, TypeSolver typeSolver) {
             if (expr instanceof VariableDeclarationExpr) {
                 for (VariableDeclarator v : ((VariableDeclarationExpr)expr).getVariables()) {
                     if (v.getName().getIdentifier().equals(name)) {
-                        JavaParserSymbolDeclaration decl = JavaParserSymbolDeclaration.localVar(v, typeSolver);
+                        ResolvedValueDeclaration decl = JavaParserSymbolDeclaration.localVar(v, typeSolver);
```

Comment:
I think this should be `JavaParserVariableDeclaration` given that we're within the `javaparsermodel` here, and `localVar` returns a `JavaParserVariableDeclaration`.

Reply:
Not that it particularly matters I suppose, given that it's returned immediately on the next line! :grinning: